download/releaseNotes/releaseNotes7.2.adoc

```diff
@@ -0,0 +1,69 @@
+= Release Notes 7.2
```

Comment:
Don't forget to adjust download/releaseNotes/index.adoc and download/upgradeRecipe/index.adoc